project/Team.py

```diff
@@ -13,12 +20,110 @@ def answer_question(self, answer):
+        self.assertEqual(self.team.currentLandmark,1,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),2,"Time Log Did Not recieve a new entry")
+        self.team.answer_question("three disks")
+        self.assertEqual(self.team.points,240,"Points did not increment correctly")
+        self.assertEqual(self.team.currentLandmark,2,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),3,"Time Log Did Not recieve a new entry")
+        self.assertEqual(self.team.timelog[2],clueTime,"Time Log Did Not Save The Correct Time")#This may not work properly
+
+#TIME PENELTY TESTS HAVE A WAIT THAT WILl CAUSE THE TEST TO RUN A UNORMALLY LONG AMOUNT OF TIME, 
+# IT IS RECCOMENDED TO HAVE THESE TESTS BE LAST IN THE SUITE
+
+    def test_answer_question_time_penalty(self):
+        self.team.clueTime = datetime.datetime.now()
+        time.sleep(16)
+        self.team.answer_question("three disks")
+        self.assertEqual(self.team.points,230,"Points did not increment correctly")
+        self.assertEqual(self.team.currentLandmark,2,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),3,"Time Log Did Not recieve a new entry")
+        self.assertEqual(self.team.timelog[2],clueTime+datetime.time(00,00,16),"Time Log Did Not Save The Correct Time") #This may not work properly
+
+    def test_answer_question_time_penalty2(self):
+        self.team.clueTime = datetime.datetime.now()
+        time.sleep(31)
+        self.team.answer_question("three disks")
+        self.assertEqual(self.team.points,210,"Points did not increment correctly")
+        self.assertEqual(self.team.currentLandmark,2,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),3,"Time Log Did Not recieve a new entry")
+        self.assertEqual(self.team.timelog[2],clueTime+datetime.time(00,00,31),"Time Log Did Not Save The Correct Time") #This may not work properly
+   
+    def test_answer_question_time_complex(self):
+        self.team.clueTime = datetime.datetime.now()
+        self.team.answer_question("two disks")
+        self.assertEqual(self.team.points,100,"Points did not increment correctly")
+        self.assertEqual(self.team.currentLandmark,1,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),2,"Time Log Did Not recieve a new entry")
+        time.sleep(18)
+        self.team.answer_question("three disks")
+        self.assertEqual(self.team.points,220,"Points did not increment correctly")
+        self.assertEqual(self.team.currentLandmark,2,"Landmark Index Did not Properly increment")
+        self.assertEqual(len(self.team.timelog),3,"Time Log Did Not recieve a new entry")
+        self.assertEqual(timelog[2],clueTime+datetime.time(00,00,18),"Time Log Did Not Save The Correct Time") #This may not work properly
+    
+
+suite = unittest.TestSuite()
```

Comment:
wrap these lines in an `if __name__ == "__main__":` so that they don't get run when imported by other files